androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/playerOverlay/Settings.kt

```diff
@@ -42,9 +44,13 @@ fun Settings(
     onDismissRequest: () -> Unit,
     onSpeedSelected: (Float) -> Unit,
     onIsLoopEnabledSelected: (Boolean) -> Unit,
+    selectedQualityTrack: TrackQualityItem,
+    qualityTracks: () -> List<TrackQualityItem>,
+    onQualityChanged: (TrackQualityItem) -> Unit,
     customization: SettingsControlsCustomization
 ) {
     val sheetState = rememberModalBottomSheetState(skipPartiallyExpanded = true)
+    val isQualitySelectorVisible = remember(qualityTracks().size) { qualityTracks().size > 2 }
```

Comment:
avoid magic numbers, I understand that 2 stands for auto + one other, right? Why don't create a variable for that?

```kotlin
// At least we should have two tracks, one for AUTO and another for the
// actual bitrate quality to be played
private const val MIN_TRACKS_SIZE = 2
```